test/e2e/app/app.go

```diff
@@ -275,6 +275,17 @@ func (app *Application) PrepareProposal(
 	return abci.ResponsePrepareProposal{BlockData: req.BlockData}
 }
 
+// ProcessProposal implements ABCI
```

Comment:
A useful addendum might also be:
```suggestion
// ProcessProposal implements part of the Application interface.
// It accepts any proposal that does not contain an invalid transaction.
```